docs/rfcs/003-open-in-atom.md

```diff
@@ -0,0 +1,137 @@
+
+This is a moment where an "Open in Atom" feature would come in handy. Effortlessly switch from looking at a line on github.com to have the same line open in Atom, ready to be edited and tested locally.
+
+Instead, Jody does the following:
+
+1. Scrolls to the top of the page
+2. Clicks on "Conversation"
+3. Scrolls to the bottom of the page
+4. Clicks on "command line instructions"
+5. Copies the `git checkout` part
+6. Opens Terminal
+7. CDs to the repo's directory
+8. Pastes the `git checkout` command
+9. Types `atom .`
+10. While Atom is opening, goes back to the diff to copy the line.
+11. In Atom, does a "Find in project" with the copied line
+12. Clicks on the found result
+13. Finally is ready to edit the line
+
+How can we improve this "start on github.com, continue in Atom" workflow? How can the dozen of steps in the above example be reduced to just 1-2 clicks?
+
+## Explanation
+
+In various places on github.com a user has the option to "Open in Atom". It's the same idea as the "Open in Desktop" that already exists.
+
+> Note: Out of simplicity, GitHub Desktop or other editors are not shown in the mockups. If multiple options are available, a dropdown could be used to make a choice.
+
+![image](https://user-images.githubusercontent.com/378023/41336470-4fe89d04-6f27-11e8-9d0d-9f5b4682db9b.png)
+
+Once a user authenticates the GitHub package in Atom with their GitHub login, an "Open in Atom" option will be available on github.com in the following places:
+
+### Repo
+
+1. Clones the repo to Atom's "Project Home" (config) directory
+2. Opens a new Atom window with the repo as its project
+
+![image](https://user-images.githubusercontent.com/378023/41266162-c59358b6-6e30-11e8-9cf9-3c420fb1c2db.png)
+
+### Pull Request
+
+1. Clones the repo (if needed)
+2. Checks out the branch of the PR
+3. Opens a new Atom window
+    - Or focuses the window if the repo already exists as an opened project
+4. Opens the GitHub panel showing the PR details
+
+![image](https://user-images.githubusercontent.com/378023/41266455-e0abfd96-6e31-11e8-9f0e-124a3adc4058.png)
+
+
+### Merge conflict
+
+1. Clones the repo (if needed)
+2. Checks out the branch
+3. Opens a new Atom window
+    - Or focuses the window if the repo already exists as an opened project
+4. Opens the Git panel (showing all conflicting files)
+5. Opens the (first) conflicting file
+6. Scrolls to the (first) conflict
```

Comment:
Related: we should also be doing this when you open a file from the Git tab, but we aren't yet; see #812.